1226-editor/editor_opens_editor_administration.feature

```diff
@@ -1,13 +1,35 @@
 Feature: Editor opens Editor Administration
 
+    In order to inform about the institution's configurations
+    As an Editor
+    I want to see institution names, codes, and institutional author 
+
+    Background:
+        Given a logged in Editor
+
+    @updated
     @TEST_NP-4153
     @364
-    Scenario: Editor opens Editor Administration
-        Given that the user is logged in as Editor
-        When they click the menu item Editor Administration
-        Then they see the page Editor Administration
-        And they see a field for Email
-        And they see a Save Button that is enabled
+    Scenario: Editor opens institutions configuration
+        When the Editor opens the institution's configuration menu item
+        Then the Editor sees one or many registered official names in Bokmål, English, Nynorsk or Northern Sámi
```

Comment:
Skjønner ikke helt hva dette betyr. "registered official names" av hva? Betyr det at alt innhold i NVA skal ha oversettelser på alle fire språk? (har bare norsk/bokmål og engelsk i dag). Eller at institusjonsnavn skal kunne ha oversettelser på alle fire språk?

Reply:
Nokon institusjonar har fire offisielle navn (NTNU har 5, også Sør-Samisk), vi må kunne ta vare på dei. Det vil komme andre scenario, der vi skal bruke dei. 
Innholdet i NVA kan ha det språk det vil ha, det er metadata registrert på 1. fane.
NVA må kunne ha oversetting av applikasjonens ledetekster og knapper på desse fire språk. Det vil komme scenario som dekker det.

Reply:
Ser vi har `en`, `nb` og `nn` for NTNU i dag https://api.dev.nva.aws.unit.no/cristin/organization/194.0.0.0